end_to_end/gemma/7b/1_test_gemma.sh

```diff
@@ -8,22 +8,36 @@
 # 2. Convert the scanned checkpoint from step 1 into unscanned checkpoint format and run more efficient decoding.
 
 # Example Usage: bash end_to_end/gemma/7b/1_test_gemma.sh
+# OR
+# export MODEL_BUCKET=/path/to/GCS/bucket/and/an/index; bash end_to_end/gemma/7b/1_test_gemma.sh
+
 set -ex
-idx=$(date +%Y-%m-%d-%H-%M)
 MODEL_VARIATION='7b'
 
 # Non-Googlers please remember to point `BASE_OUTPUT_DIRECTORY` to a GCS bucket that you own, this bucket will store all the files generated by MaxText during a run
 export BASE_OUTPUT_DIRECTORY=gs://runner-maxtext-logs
 # After downloading checkpoints, copy them to GCS bucket at $CHKPT_BUCKET \
 # Please use seperate GCS paths for uploading model weights from kaggle ($CHKPT_BUCKET) and MaxText compatible weights ($MODEL_BUCKET).
-# Non-Googlers please remember to point these variables to GCS buckets that you own, this script uses internal buckets for testing.
+
 export CHKPT_BUCKET=gs://maxtext-gemma/flax
-export MODEL_BUCKET=gs://maxtext-gemma
-JAX_PLATFORMS=cpu python MaxText/convert_gemma_chkpt.py --base_model_path ${CHKPT_BUCKET}/${MODEL_VARIATION} --maxtext_model_path ${MODEL_BUCKET}/${MODEL_VARIATION}/${idx} --model_size ${MODEL_VARIATION}
-echo "Writen MaxText compatible checkpoint to ${MODEL_BUCKET}/${MODEL_VARIATION}/${idx}"
+
+if [ -z "${MODEL_BUCKET}" ]; then
+    # Non-Googlers please remember to point these variables to GCS buckets that you own, this script uses internal buckets for testing.
+    export MODEL_BUCKET=gs://maxtext-gemma
+    idx=$(date +%Y-%m-%d-%H-%M)
+    
+else
+    idx=$(awk -F'/' '{print $NF}' <<<"${MODEL_BUCKET}")
```

Comment:
what is this AWK doing? Seems unwieldy, maybe you can explain the problem (ideally in some message about the commit?)

Reply:
Thanks Rafi - I added the explanation in the description of this PR.

Reply:
Ok that is super helpful.

Is there a way we could not have thing branching? Like do we really to preserve multiple options here?

Reply:
Did you mean we remove the two options of connecting the two testing scripts, and only have the option of this?
(we need this option of connecting the two scripts for our internal testing though )

```
export MODEL_BUCKET=/path/to/GCS/bucket/and/an/index; bash end_to_end/gemma/7b/1_test_gemma.sh
export MODEL_BUCKET=/path/to/GCS/bucket/and/an/index; bash end_to_end/gemma/7b/2_test_gemma.sh
```

Reply:
Yes

Reply:
(Or maybe focus on a version that doesn't require parsing so you apss two args.

Reply:
I can definitely make passing `MODEL_BUCKET` as the only option 

Could we pass both a GCS path as well as an additional index from airflow? @jonb377 and @RissyRan

Reply:
Followed up with @RissyRan - it is possible, it would need some modifications to our airflow+XLML tests code.

Reply:
@rwitten I think the much simpler and cleaner solution will be to have one `GCS path` shared between the two scripts and it would be the `BASE_OUTPUT_DIRECTORY`, and write everything there: for e.g.,  `scanned ckpt`, `unscanned ckpt`, and all outputs of `train` and `decode`. This shared `GCS path` will be unique since it will have the datetime inside it and will be generated from airflow.

I updated the code, please take a look.

cc: @RissyRan @jonb377 @khatwanimohit